src/components/SearchMap/SearchMap.js

```diff
@@ -145,11 +145,8 @@ export class SearchMapComponent extends Component {
     // When changing from default map provider to Google Maps, you should use the following
     // component instead of SearchMapWithMapbox:
     //
-    // <SearchMapWithGoogleMap
-    //   containerElement={
-    //     <div id="search-map-container" className={classes} onClick={this.onMapClicked} />
-    //   }
-    //   mapElement={<div className={mapRootClassName || css.mapRoot} />}
+    // <SearchMapWithGoogleMaps
```

Comment:
It might be useful to briefly mention the component name change in the PR description.